packages/client/tests/functional/_utils/getTestSuitePlan.ts

```diff
@@ -64,21 +105,45 @@ function shouldSkipProvider(
     return true
   }
 
+  // if the client doesn't support the provider, skip
+  if (testCliMeta.dataProxy && provider === 'sqlite') {
+    return true
+  }
+
+  // if the provider is not included, skip
   if (includedProviders && !includedProviders.includes(provider)) {
     return true
   }
 
-  if (clientMeta.dataProxy && provider === 'sqlite') {
+  // if the provider is excluded, skip
+  if (excludedProviders.includes(provider)) {
     return true
   }
 
-  return excludedProviders.includes(provider)
+  // if there is a flavor to run and it's not included, skip
+  if (flavor && !includedProviderFlavors?.includes(flavor)) {
+    return true
+  }
+
+  // if there is a flavor to run and it's excluded, skip
+  if (flavor && excludedProviderFlavors.includes(flavor)) {
+    return true
+  }
```

Comment:
We have two mechanism to skip tests, lack of inclusion and explicit exclusion, Is it a correct assumption that by default, all flavors are included, and there's no need to specify all in the list, per specific suite?

If so, wouldn't it make sense to only have a list of exclusions?

Reply:
I think it would make sense to use exclusion lists only, as Miguel says.

Reply:
It works the other way around. By default, no flavors are included. As such, the exclusion could be removed, and I mostly kept it to keep the code isomorphic.